chunky/src/java/se/llbit/chunky/renderer/scene/SkyCache.java

```diff
@@ -16,88 +16,101 @@
  */
 package se.llbit.chunky.renderer.scene;
 
+import static java.lang.Math.PI;
+
+import java.util.concurrent.ForkJoinPool;
+import java.util.stream.IntStream;
 import org.apache.commons.math3.util.FastMath;
+import se.llbit.chunky.PersistentSettings;
 import se.llbit.math.ColorUtil;
 import se.llbit.math.QuickMath;
 import se.llbit.math.Ray;
 import se.llbit.math.Vector3;
 
-import static java.lang.Math.PI;
-
 public class SkyCache {
+
   // Sky texture array, HSV colors
   private double[][][] skyTexture;
 
   // Default resolution is 128x128. Should be enough for most simulated skies.
   private int skyResolution = 128;
 
   private SimulatedSky simSky;
-  private Sky sky;
 
   /**
-   * An on-the-fly sky cache. Automatically calculates sky colors as they are requested and caches them. Any repeat
-   * requests will pull from the cache.
-   *
-   * Default cache size is 1024x1024 with bilinear interpolation which seems to work well for all simulated sky
-   * modes.
+   * An on-the-fly sky cache. Automatically calculates sky colors as they are requested and caches
+   * them. Any repeat requests will pull from the cache.
+   * <p>
+   * Default cache size is 1024x1024 with bilinear interpolation which seems to work well for all
```

Comment:
Might want to change this to the correct value of `128x128` for anyone reading the javadocs.

Reply:
And maybe remove `on-the-fly` and mention that it is precalculated on creation and reset do discourage needlessly resetting it.